From 268580a83c75274a5bee7fb9b03340d683de9190 Mon Sep 17 00:00:00 2001
From: Denis Ollier <larchunix@users.noreply.github.com>
Date: Thu, 12 Apr 2018 21:31:30 +0200
Subject: [PATCH] Don't truncate version string when building from a git tag

32 characters are not enough to store the version string with Remmina's
current versioning scheme.

For instance, "1.2.0-rcgit-29 (git v1.2.0-rcgit-29)" uses 36 characters.
---
 remmina/src/remmina_about.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/remmina/src/remmina_about.c b/remmina/src/remmina_about.c
index d66d8c1f..5b8380fb 100644
--- a/remmina/src/remmina_about.c
+++ b/remmina/src/remmina_about.c
@@ -45,7 +45,7 @@
 void remmina_about_open(GtkWindow *parent)
 {
 	TRACE_CALL(__func__);
-	static gchar version[32];
+	static gchar version[40];
 
 	g_snprintf(version, sizeof(version), "%s (git %s)", VERSION, REMMINA_GIT_REVISION);
 	GtkBuilder *builder = remmina_public_gtk_builder_new_from_file("remmina_about.glade");
-- 
2.17.0

